tableauserverclient/server/endpoint/jobs_endpoint.py

```diff
@@ -11,7 +11,20 @@ def baseurl(self):
         return "{0}/sites/{1}/jobs".format(self.parent_srv.baseurl, self.parent_srv.site_id)
 
     @api(version='2.6')
-    def get(self, job_id):
+    def get(self, job_id=None, req_options=None):
+        # Backwards Compatibility fix until we rev the major version
+        if job_id is not None and isinstance(job_id, basestring):
+            import warnings
+            warnings.warn("Jobs.get(job_id) is deprecated, update code to use Jobs.get_by_id(job_id)")
+            return self.get_by_id(job_id)
+        self.parent_srv.assert_at_least_version('3.1')
```

Comment:
I see now, you need to call this assert to check that it's > 3.1 and also keep the > 2.6 for old back compat. I need to think about this, maybe it's better to have another decorator behavior_changed_in... or something like that. 

But this seems fine for now